internal/tspath/path.go

```diff
@@ -533,13 +533,40 @@ func GetCanonicalFileName(fileName string, useCaseSensitiveFileNames bool) strin
 // Rest special characters are either already in lower case format or
 // they have corresponding upper case character so they dont need special handling
 
-func ToFileNameLowerCase(fileName string) string {
+func ToFileNameLowerCase(s string) string {
```

Comment:
This is a nit, but I'd prefer if the parameter were still named `fileName`.

Reply:
https://github.com/microsoft/typescript-go/pull/1558/commits/dcb78e8afda3c11175a704b9a3194029cd125f7e